lib/charms/opensearch/v0/opensearch_tls.py

```diff
@@ -302,6 +327,7 @@ def _request_certificate_renewal(
         subject = self._get_subject(cert_type)
         organization = self.charm.opensearch_peer_cm.deployment_desc().config.cluster_name
         new_csr = generate_csr(
+            add_unique_id_to_subject_name=False,
```

Comment:
Could you explain this?

Reply:
I found while investigating another task, will copy the answer from [the PR](https://github.com/canonical/opensearch-operator/pull/408):

```
[...] a bugfix for renewing the expiring certificates, or better creating the certificate signing requests: When initially creating them, they do not include unique-id in the subject. But on renewal this is included, which causes the subject of the renewed certificate to be different from the initial one.
```

Even though the `issuer` will change on CA-rotation, subjects should stay the same if the unit itself didn't change.